microservices/cmslitemetadata_to_redshift/ddl/cmslite.metadata.sql

```diff
@@ -0,0 +1,200 @@
+)  
+DISTSTYLE EVEN;
+
+CREATE TABLE IF NOT EXISTS cmslite.dcterms_subjects (
+    "id"      BIGINT		ENCODE LZO	NOT NULL,
+    "value"   VARCHAR(255)  ENCODE ZSTD	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+CREATE TABLE IF NOT EXISTS cmslite.dcterms_languages (
+    "id"      BIGINT		ENCODE LZO	NOT NULL,
+    "value"   VARCHAR(255)  ENCODE ZSTD	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+-- SY: Table no longer in cmslite schema
+/* CREATE TABLE IF NOT EXISTS cmslite.security_classifications (
+    "id"      BIGINT          NOT NULL,
+    "value"   VARCHAR(255)     NOT NULL
+); */
+
+-- SY: Table no longer in cmslite schema
+/* CREATE TABLE IF NOT EXISTS cmslite.security_labels (
+    "id"      BIGINT          NOT NULL,
+    "value"   VARCHAR(255)     NOT NULL
+); */
+
+CREATE TABLE IF NOT EXISTS cmslite.audiences (
+    "id"      BIGINT		ENCODE LZO	NOT NULL,
+    "value"   VARCHAR(255)  ENCODE ZSTD	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+-- SY: Added this table to reflect cmslite_gdx.json
+CREATE TABLE IF NOT EXISTS cmslite.creators (
+    "id"      BIGINT		ENCODE LZO	NOT NULL,
+    "value"   VARCHAR(255)  ENCODE ZSTD	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+-- LOOKUP TABLES (MANY-TO-MANY)
+
+-- SY: dbtables_metadata
+
+CREATE TABLE IF NOT EXISTS cmslite.metadata_content_types (
+    "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+CREATE TABLE IF NOT EXISTS cmslite.metadata_subject_categories (
+    "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+CREATE TABLE IF NOT EXISTS cmslite.metadata_subjects (
+    "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+CREATE TABLE IF NOT EXISTS cmslite.metadata_languages (
+     "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+-- SY: Table no longer in cmslite schema
+/* CREATE TABLE IF NOT EXISTS cmslite.metadata_security_classifications (
+    "node_id"   VARCHAR(255)    NOT NULL,
+    "id"        BIGINT          NOT NULL
+); */
+
+-- SY: Table no longer in cmslite schema
+/* CREATE TABLE IF NOT EXISTS cmslite.metadata_security_labels (
+    "node_id"   VARCHAR(255)    NOT NULL,
+    "id"        BIGINT          NOT NULL
+); */
+
+CREATE TABLE IF NOT EXISTS cmslite.metadata_audiences (
+    "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+-- SY: Added this table to reflect cmslite_gdx.json
+CREATE TABLE IF NOT EXISTS cmslite.metadata_creators (
+    "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+-- METADATA TABLE FOR NODE_ID RECORDS
+-- NODE_ID should be primary key, however RedShift doesn't seem to enforce this.
```

Comment:
Redshift can enforce that, but we haven't defined node_id as a primary key in the schema, so it doesn't.

If you want to propose that change, let's talk about it in the ticket as an improvement; maybe to do as a later piece of work (we'd need to QA the change).

please remove this comment from the code.

Reply:
This is a note from the original SQL Fiddle. I don't need to propose any changes, I will remove the comment.